source/simulator/solver.cc

```diff
@@ -253,25 +253,91 @@ namespace aspect
       S_block_tolerance(S_block_tolerance)
     {}
 
-    template <class PreconditionerA, class PreconditionerMp>
+    template <class PreconditionerA, class SInvOperator>
     unsigned int
-    BlockSchurPreconditioner<PreconditionerA, PreconditionerMp>::
+    BlockSchurPreconditioner<PreconditionerA, SInvOperator>::
     n_iterations_A() const
     {
       return n_iterations_A_;
     }
 
-    template <class PreconditionerA, class PreconditionerMp>
+    template <class PreconditionerA, class SInvOperator>
     unsigned int
-    BlockSchurPreconditioner<PreconditionerA, PreconditionerMp>::
+    BlockSchurPreconditioner<PreconditionerA, SInvOperator>::
     n_iterations_S() const
     {
-      return n_iterations_S_;
```

Comment:
did you delete the member variable n_iterations_S_? I don't see it in the patch.